site/en/models.md

```diff
@@ -114,6 +120,40 @@ This will download and store the resource in the [proper location](#where-is-the
 ocrd-tesserocr-recognize -P model mymodel
 ```
 
+### Models and Docker
+
+If you are using OCR-D with Docker, we recommend keeping all downloaded resources in a persistent host directory,
+separate of the `ocrd/*` Docker container and data directory, and mounting that
+resource directory into a specific path in the container alongside the data directory.
+The host resource directory can be empty initially. Each time you run the Docker container,
+your processors will access the host directory to resolve resources, and you can download
+additional models into that location using `ocrd resmgr`.
+
+The following will assume (without loss of generality) that your host-side data
+path is under `./data`, and the host-side resource path is under `./models`:
+
+To download models to `./models` in the host FS and `/usr/local/share/ocrd-resources` in Docker:
+
+```sh
+docker run --user $(id -u) \
+  --volume $PWD/models:/usr/local/share/ocrd-resources \
+ocrd/all \
+ocrd resmgr download ocrd-tesserocr-recognize eng.traineddata\; \
+ocrd resmgr download ocrd-calamari-recognize default\; \
+...
+```
+
+To run processors, as usual do:
+
+```sh
+docker run --user $(id -u) --workdir /data \
+  --volume $PWD/data:/data \
+  --volume $PWD/models:/usr/local/share/ocrd-resources \
```

Comment:
Is that still correct for `ocrd-tesserocr-*`?

```
docker run --rm -it ocrd/all:latest ocrd-tesserocr-recognize -D
/usr/local/share/tessdata/
```

Reply:
OK, then let's merge this first and track the ocrd_tesserocr question in an issue.

Reply:
*sigh* this question keeps being asked...

https://github.com/OCR-D/ocrd_all/blob/b36cec819c67ccbdbed8033007ea27596e02f972/Makefile#L724
https://github.com/OCR-D/ocrd_all/blob/b36cec819c67ccbdbed8033007ea27596e02f972/Makefile#L756
https://github.com/OCR-D/ocrd_all/blob/b36cec819c67ccbdbed8033007ea27596e02f972/Dockerfile#L32-L33

so – no, /usr/local/share/ocrd-resources will be ignored in ocrd_tesserocr when installed via ocrd_all, the module resource location is /usr/local/share/tessdata, as @kba wrote.

Reply:
So we do still have a problem with our model volume logic here. Modules like ocrd_tesserocr or workflow-configuration (`ocrd-page-transform` ...) want to have their stuff under /usr/local/share/XYZ, while others use /usr/local/share/ocrd-resources.

Everything is ok for the preinstalled resources (tool json files for bashlib processors, preset files for ocrd-page-transform, minimal models for Tesseract). But as soon as you want to install **additional** models **persistently**, we cannot offer anything ATM.

(So this is not just about the right kind of recipe covering the volume mapping, but accomodating a single `module` location inside the Docker image – because it is prebuilt – with persistent updates we usually do via `data` location...)